DeepSpeech.py

```diff
@@ -79,32 +79,65 @@ def dense(name, x, units, dropout_rate=None, relu=True):
 
 
 def rnn_impl_lstmblockfusedcell(x, seq_length, previous_state, reuse):
-    # Forward direction cell:
-    fw_cell = tf.contrib.rnn.LSTMBlockFusedCell(Config.n_cell_dim, reuse=reuse)
+    with tf.variable_scope('cudnn_lstm/rnn/multi_rnn_cell/cell_0'):
```

Comment:
I'm not sure I get what this is supposed to do precisely. You place `LSTMBlockCell` below a `cudnn_lstm` ? Or you **replace** ?